app/jobs/sms_sender_number_change_job.rb

```diff
@@ -1,4 +1,6 @@
 class SmsSenderNumberChangeJob < ActiveJob::Base
+  include Rails.application.routes.url_helpers
+  Rails.application.routes.default_url_options[:host] = Figaro.env.mailer_domain_name
   queue_as :sms
```

Comment:
Is this necessary even though we have this in our `config/environments/*.rb` files?
```ruby
config.action_mailer.default_url_options = {
  host: Figaro.env.domain_name,
  protocol: 'https'
}
```